parsons/databases/mysql/create_table.py

```diff
@@ -0,0 +1,159 @@
+import ast
+import petl
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class MySQLCreateTable():
+
+    def __init__(self):
+
+        pass
+
+    def data_type(self, val, current_type):
+        # Evaluate the MySQL data type of a given value. Then return
+        # the data type of the value, while also weighing what the
+        # current type is.
+
+        # Stop if the current type is already a varchar
+        if current_type == 'varchar':
+            return current_type
+
+        # Convert to string to reevaluate data type
+        try:
+            t = ast.literal_eval(str(val))
+        except ValueError:
```

Comment:
Can you outline the conditions for these two errors? Is one covering the case where the incoming value is already a string, and the other covering the case where that string is empty?